vms/platformvm/vm_test.go

```diff
@@ -63,30 +63,23 @@ import (
 	txexecutor "github.com/ava-labs/avalanchego/vms/platformvm/txs/executor"
 )
 
-var (
-	latestForkTime = genesistest.ValidateEndTime.Add(-5 * configtest.MinStakingDuration)
```

Comment:
no global latestForkTime anymore. So we don't have to worry to reset it across UTs